src/coreclr/tools/superpmi/superpmi/superpmi.cpp

```diff
@@ -159,12 +159,13 @@ static void PrintDiffsCsvRow(
     fw.Printf("%d,%u,", context, contextSize);
     fw.PrintQuotedCsvField(baseRes.CompileResults->MethodFullName == nullptr ? "" : baseRes.CompileResults->MethodFullName);
     fw.Printf(
-        ",%s,%s,%s,%s,%s,%u,%u,%lld,%lld",
+        ",%s,%s,%s,%s,%s,%u,%u,%u,%u,%lld,%lld",
         baseRes.CompileResults->TieringName == nullptr ? "" : baseRes.CompileResults->TieringName,
         ResultToString(baseRes.Result), ResultToString(diffRes.Result),
         baseRes.IsMinOpts ? "True" : "False",
         hasDiff ? "True" : "False",
         baseRes.NumCodeBytes, diffRes.NumCodeBytes,
+        (uint32_t)baseRes.NumGCInfoBytes, (uint32_t)diffRes.NumGCInfoBytes,
```

Comment:
There is no need to add this manually, all metrics are automatically included

Reply:
Maybe we should add hot/cold/total code size as metrics, then get rid of `NumCodeBytes` here (as well as getting rid of the `NumGCInfoBytes` that I added)?

Reply:
Yeah, we could definitely move more of this stuff into metrics now.